resources/main.css

```diff
@@ -1,3 +1,13 @@
+a {
+    color: white;
+    font-family: Roboto,sans-serif;
+}
+
+button {
```

Comment:
No, I don't think you should keep the styling in `main.css` because it is for a single button in `options.html`. Then it is not a real general thing.